libs/cli/langgraph_cli/config.py

```diff
@@ -990,26 +990,25 @@ def _update_graph_paths(
                 raise IsADirectoryError(f"Local module must be a file: {resolved}")
             else:
                 for path in local_deps.real_pkgs:
+                    
                     if resolved.is_relative_to(path):
-                        container_path = (
-                            pathlib.Path("/deps")
-                            / path.name
-                            / resolved.relative_to(path)
-                        )
-                        module_str = container_path.as_posix()
+                        container_subpath = resolved.relative_to(path).as_posix()
+                        container_path = f"/deps/{path.name}/{container_subpath}"
```

Comment:
Could you avoid `f` strings here and rely more on Pathlib functionality together with explicit as_posix() or str(path) to get native OS representation